tedana/workflows/ica_reclassify.py

```diff
@@ -232,16 +258,72 @@ def _parse_manual_list(manual_list):
         manual_nums = manual_list
     else:
         raise ValueError(
-            f"_parse_manual_list expected integers or a filename, but the input is {manual_list}"
+            "_parse_manual_list_int expected integers or a filename, "
+            f"but the input is {manual_list}"
         )
 
     return manual_nums
 
 
+def _parse_manual_list_str(manual_list):
```

Comment:
It might be good to write a unit test for this function. It would be simpler to just test this function directly than to incorporate the test into the integration tests for ica_reclassify.